proto/quotacontrol.gen.ts

```diff
@@ -1170,7 +1259,7 @@ const webrpcErrorByCode: { [code: number]: any } = {
   [1005]: AbortedError,
   [1006]: GeoblockedError,
   [1007]: RateLimitedError,
-  [1100]: ProjectNotFoundError,
+  [1008]: ProjectNotFoundError,
```

Comment:
Did we change this error code for any reason? If other projects rely on this value as of today, this may cause issues.

Reply:
It is now defined in AuthControl now, since it's used also there, so I had to remove it here.